openapi/multi-yaml/index.yaml

```diff
@@ -79,223 +79,141 @@ tags:
-                  "brand": "mastercard",
-                  "name": "Tifa Lockhart",
-                  "token": "pm_autoclonedxyz789"
-                }
-              }
-            }
-          }
-        }
-      }
+      #### Via API
+      ```
+      // Create checkout
+      const checkout = await fetch('https://api.justifi.ai/v1/checkouts', {
+        method: 'POST',
+        headers: {
+          'Authorization': `Bearer ${token}`,
+          'Sub-Account': processingSubAccountId,  // Processing sub account
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          "amount": 1799,
+          "description": "Example item",
+          "payment_method_group_id": "pmg_walletGroupId" // Group from wallet account
+        })
+      });
+
+      // Complete checkout with wallet payment method
+      const completion = await fetch(`https://api.justifi.ai/v1/checkouts/${checkoutId}/complete`, {
+        method: 'POST',
+        headers: {
+          'Authorization': `Bearer ${token}`,
+          'Sub-Account': processingSubAccountId,  // Processing sub account
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          "payment_token": "pm_walletPaymentMethodToken"
+        })
+      });
       ```
 
-      Note that when completing the checkout with a wallet payment method token, the system automatically creates a cloned payment method for the sub-account as shown in the response's payment_method.token field. 
-      This same token will be retrieved for subsequent payment requests for the same Sub-Account passing the same wallet token
-
-      3. Optionally organize payment methods into groups using the Payment Method Groups API
+      ## Updating Wallet Payment Methods
+      - Updates should always be made to the payment method in the platform wallet account
+      - Any changes made to the platform wallet payment method automatically propagate to all cloned payment methods across sub accounts
+      - Updates include:
+        - Card expiration updates
+        - Payment method metadata
+      - The system maintains consistency by:
+        - Automatically syncing updates to all cloned versions
+
+      ## Important Notes
+      - Header Requirements:
+        - Use **platform wallet account ID** for:
+          - Creating/managing payment methods
+          - Creating/managing payment method groups
+        - Use **processing sub account ID** for:
+          - Creating checkouts
+          - Completing payments
+      - The system automatically:
+        - Validates wallet payment method access
+        - Creates payment method clones for processing sub accounts
+        - Returns new sub account specific tokens in responses
+      - All sub accounts must be on the same platform as the platform wallet account
+
+      For complete details on specific endpoints, see:
+      - [Checkout via Component](http://localhost:3001/api-spec#tag/Checkout-via-Component)
```

Comment:
This one needs the producation url: https://docs.justifi.tech/api-spec#tag/Checkout-via-Component